### ISSUE TITLE
Not providing a getter causes downstream error

### ISSUE THREAD

**Issue by will-holley** [graphile/persisted-operations#1]
### Summary

See title.

### Steps to reproduce

```javascript
!options.persistedOperationsGetter && !options.persistedOperations && !options.persistedOperationsDirectory
```

### Actual results

[This clause](https://github.com/graphile/postgraphile/blob/3b01d529e7fdc79e14a0bde0bf133e963ecdc088/src/postgraphile/http/createPostGraphileHttpRequestHandler.ts#L691) is triggered.

### Possible Solution

- Bubble the error upstream by replacing [`null assignment`](https://github.com/graphile/persisted-operations/blob/3aff08dfd51104e78bdeddf4011895e95ab647c7/src/index.ts#L162) with `throw new Error('No getter function was specified')`.

**Comment by benjie** [comment]
This is currently by design, it'd be nice if we could replace the actual error message, but because we're hooked into third party libraries e.g. for the websocket transport we cannot reliably throw an error at this point. Our best option is to invalidate the request.

https://github.com/graphile/persisted-operations/blob/3aff08dfd51104e78bdeddf4011895e95ab647c7/src/index.ts#L183-L186

https://github.com/graphile/persisted-operations/blob/3aff08dfd51104e78bdeddf4011895e95ab647c7/src/index.ts#L255

**Comment by benjie** [comment]
I've made the server throw an error on start-up if no getter is configured though: https://github.com/graphile/persisted-operations/commit/3ecc8c00716fcfba8fd73eb320bdc2ad94b21407

Released in v0.0.2

**Comment by will-holley** [comment]
Thanks for the prompt response & address! A modicum of sanity has been introduced into the world.
